draft-ietf-rtcweb-jsep.xml

```diff
@@ -2764,14 +2767,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           "fmt" fields MUST be set to exactly match the fields in the
           offer.</t>
 
-          <t>Within the data m= section, the "a=mid", "a=ice-ufrag",
-          "a=ice-pwd", "a=candidate", "a=fingerprint", "a=dtls-id",
-          and "a=setup" lines MUST be included under the conditions described above,
+          <t>Within the data m= section, an "a=mid" line MUST be
+          generated and included as described above,
           along with an "a=fmtp:webrtc-datachannel" line and an
-          "a=sctp-port" line referencing the SCTP port number as
-          defined in
-          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section
-          4.1.</t>
+          "a=sctp-port" line referencing the SCTP port number as defined in
+          <xref target="I-D.ietf-mmusic-sctp-sdp"></xref>, Section 4.1.
+          If the m= section either has a unique address or is associated
```

Comment:
For the answer, the terminology used above is "not bundled into another m= section", which sounds clearer to me.